### PR TITLE
Closes #4 (usability)

### LINKED ISSUE #4: Usability sucks
The "how to use" section in the README is an 11 chapter tome of incantations. Ideally, all functionality can be rolled into a single Rust binary

### PR DESCRIPTION
**Pull request by laudiacay**
no more opt dependency
no more rustfile dependency
no more weird bash scripts
no more usability issues
only a rust binary :)

### CODE REVIEW

**Comment by laudiacay** [comment]
although the tests aren't CI'd yet, it passes them all when RUSTC_LLVM_VERSION \in {9.*, 10.*}

File: Cargo.toml [hunk 1]
```diff
@@ -13,3 +13,7 @@ structopt = "0.3.12"
 syn = { version = "1.0", features = ["full", "visit"] }
 walkdir = "2.2"
 regex = "1"
+llvm-ir = "0.5.3"
+anyhow = "1"
+rustc-demangle = "0.1"
+glob = "0.3"
```
Comment (woodruffw) [inline Cargo.toml:19]: Nitpick: Trailing newline?

Comment (laudiacay) [reply Cargo.toml]: added

**Comment by woodruffw** [comment]
LGTM! We should get the tests in to the CI with those LLVM versions, although breaking that into another PR sounds fine to me 🙂

**Pull request merged**